modules/post/multi/gather/pidgin_cred.rb

```diff
@@ -85,6 +89,7 @@ def enum_users_unix
       home = '/home/'
     end
 
+    # @todo use Msf::Post::File
```

Comment:
Just clarifying if these were here for you to circle back to, or just calling out that `Msf::Post::File` could be used in place of these for future travelers?

Reply:
I have no intention of resolving these `@todo` comments specifically; however, line 190 of my TODO list for Metasploit states: "review all `TODO` and `@todo` comments". At the current rate, presuming nothing gets pushed to the top of my TODO stack, I will review these comments in ~15 years.

Reply:
Works for me. Looking forward to that PR 👴